discovery-provider/src/api/v1/users.py

```diff
@@ -1081,7 +1319,9 @@ def get(self, id: str):
 )
 
 
-@full_ns.route("/<string:id>/supporting/<string:supported_user_id>")
+@full_ns.route(
```

Comment:
should we have a non-full route here as well? @rickyrombo

Reply:
Yeah, we should